src/synapse/workload.py

```diff
@@ -366,6 +387,41 @@ def enable_allow_public_rooms_over_federation(container: ops.Container) -> None:
         raise WorkloadError(str(exc)) from exc
 
 
+def _create_ip_range_whitelist(ip_range_whitelist: str) -> list[str]:
+    """Format IP range whitelist.
+
+    Args:
+        ip_range_whitelist: ip_range_whitelist configuration.
+
+    Returns:
+        IP range whitelist as expected by Synapse or None.
+    """
+    return [item.strip() for item in ip_range_whitelist.split(",")]
```

Comment:
A minor question, would this require to be a list, or could immutable tuple suffice?

Reply:
Since the values will not be modified, it could be a tuple. I'll modify that in this PR https://github.com/canonical/synapse-operator/pull/119
thanks!